charts/schema-registry/values.yaml

```diff
@@ -0,0 +1,56 @@
+# /**
+#  * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License");
+#  * you may not use this file except in compliance with the License.
+#  * You may obtain a copy of the License at
+#  *
+#  *     http://www.apache.org/licenses/LICENSE-2.0
+#  */
+
+## Default values for schema-registry.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+replicas: 2
+
+image:
+  repository: devops-repo.isus.emc.com:8116/nautilus/pravega-schema-registry
```

Comment:
This URL is for internal and proprietary repo. We shouldn't be exposing it here.

Reply:
changed it